src/common/components/CopyButton.tsx

```diff
@@ -12,3 +14,71 @@ export const CopyButton = ({
     </ClipboardRoot>
   );
 };
+
+export const CopyButtonNew = ({
```

Comment:
maybe CopyButtonRedesign for ease of catching when we replace the old one

Reply:
done